app/components/header/header.component.html

```diff
@@ -363,8 +369,8 @@ <h4 class="modal-title">Info</h4>
         <input type="text" id="origin-path" style=" width: 554px !important" placeholder="https://github.com/ORIGINAL_OWNER/ORIGINAL_OWNER_REPOSITORY.git">
       </div>
       <div class="modal-footer">
-        <button type="button" class="btn btn-primary" data-dismiss="modal" onClick="fetchFromOrigin()">Confirm</button>
-        <button type="button" class="btn btn-primary" data-dismiss="modal">Close</button>
+        <button type="button" class="btn btn-primary" data-dismiss="modal" onClick="fetchFromOrigin()">test</button>
+        <!-- <button type="button" class="btn btn-primary" data-dismiss="modal">Cancel</button> -->
```

Comment:
Is this related to your change?

Reply:
Yeah I renamed the "close" button but then I just decided to remove it cause there's already an x in the corner. I'll take this out

Reply:
Sorry this was a mistake! I was testing on the wrong modal, thanks for pointing this out

Reply:
It looks like this modal is from someone previously attempting to add the sync functionality. I deleted it in the latest commit cause it wasn't being used anywhere.